src/main/java/xbot/common/injection/BaseComponent.java

```diff
@@ -0,0 +1,27 @@
+package xbot.common.injection;
+
+import javax.inject.Named;
+
+import xbot.common.command.SmartDashboardCommandPutter;
+import xbot.common.controls.sensors.XTimerImpl;
+import xbot.common.injection.wpi_factories.DevicePolice;
+import xbot.common.logging.RobotAssertionManager;
+import xbot.common.properties.ITableProxy;
+import xbot.common.properties.PermanentStorage;
+import xbot.common.properties.XPropertyManager;
+
+public abstract class BaseComponent {
```

Comment:
This classname was a little confusing to me at first, because it's not really a single 'component', what do you think about `BaseComponents`?